functions/tx-manager_dashboard/main.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import unicode_literals
+import logging
+import json
+from aws_tools.dynamodb_handler import DynamoDBHandler
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+# noinspection PyUnusedLocal
+def handle(event, context):
+    """
+    Triggered by user entering one of the following endpoints into a web browser
+    https://api.door43.org/dashboard and 
+    https://test-api.door43.org/dashboard
+
+    :param dict event:
+    :param context:
+    """
+    global logger
+    logger.info( "starting" )
```

Comment:
Is there a reason you add spaces between parentheses? For the sake of consistency with other parts of the codebase, I think it would be best not to include them (omitting spaces is also in agreement with the PEP 8 Python conventions)